msteams-platform/bots/how-to/format-ai-bot-messages.md

```diff
@@ -0,0 +1,459 @@
+---
+title: Format AI bot messages
+description: Learn how to format your AI-based bot with an AI label, sensitivity labels, citations, and feedback buttons.
+ms.topic: conceptual
+ms.localizationpriority: medium
+---
+
+# Format AI bot messages
+
+If you're building a conversational bot, you can now leverage features such as citations and feedback buttons that enable better user engagement. For AI-powered bots, you can now add AI label and sensitivity label that can help you build trust and transparency with users.
+
+Here's a quick overview on each feature that you can integrate to your bot message:
+
+| Feature | Bot | AI bot |
+| --- | :---: | :---: |
+| [AI label](#ai-label) that enables user to identify that the message was generated using AI. | ❌ | ✔️ |
+| [Sensitivity label](#sensitivity-label) that enables user to understand the confidentiality of the bot's message. | ❌ | ✔️ |
+| [Citations](#citations) that enables user to refer to the source of the bot's message through in-text citations and a list of references. | ✔️ | ✔️ |
+| [Feedback buttons](#feedback-buttons) that enables user to provide positive or negative feedback based on their experience. | ✔️ | ✔️ |
+
+:::image type="content" source="../../assets/images/bots/ai-bot-message.png" border="false" alt-text="Screenshot shows the AI-generated bot message with the respective AI bot message features.":::
+
+> [!NOTE]
+>
+> AI label, sensitivity label, citations, and feedback buttons:
+>
+> * Are available in [public developer preview](../../resources/dev-preview/developer-preview-intro.md).
+> * Aren't supported in message extensions.
+
+## AI label
+
+AI label added to your bot message specifies that the message is generated by AI. AI bots use large language models (LLMs) that are generally reliable, however, it's crucial to add an AI label to maintain user's trust on your bot.
```

Comment:
This feels like a non sequitur.... How is adding an AI label crucial to maintaining the user's trust? We need to explain.

Reply:
Content updated. Removed the point about user trust and replaced it with messaging that adding an AI label helps prevent any confusion about the source of the information.